.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,63 @@
+name: PULL REQUEST
```

Comment:
Is this file a copy of the one we have in the sdk repo? Any way to have them in sync?

Reply:
No, this workflow is different. the workflow leverages different setup (building the integrations apps in matrix and so on). Thus it can't be backpropagated to our main repo.